packages/primitives/calendar/src/calendar-cell-trigger.directive.ts

```diff
@@ -0,0 +1,200 @@
+import { AfterViewInit, computed, Directive, ElementRef, inject, input } from '@angular/core';
+import { DateValue, getLocalTimeZone, isSameDay, isSameMonth, isToday } from '@internationalized/date';
+import * as kbd from '@radix-ng/primitives/core';
+import { getDaysInMonth } from '@radix-ng/primitives/core';
+import { PrimitiveElementController, usePrimitiveElement } from './usePrimitiveElement';
+import { injectCalendarRootContext } from './сalendar-сontext.token';
+
+@Directive({
+    selector: '[rdxCalendarCellTrigger]',
+    exportAs: 'rdxCalendarCellTrigger',
+    host: {
+        role: 'button',
+        '[attr.data-rdx-calendar-cell-trigger]': '""',
+        '[attr.tabindex]': 'isFocusedDate() ? 0 : isOutsideView() || isDisabled() ? undefined : -1',
+        '[attr.data-value]': 'day()?.toString()',
+        '[attr.data-today]': 'isDateToday() ? "" : undefined',
+        '[attr.data-outside-view]': 'isOutsideView() ? "" : undefined',
+        '[attr.data-selected]': 'isSelectedDate() ? "" : undefined',
+
+        '(click)': 'onClick()',
+
+        '(keydown)': 'onArrowKey($event)'
+    }
+})
+export class RdxCalendarCellTriggerDirective implements AfterViewInit {
+    private readonly rootContext = injectCalendarRootContext();
+    private readonly elementRef = inject(ElementRef<HTMLElement>);
+
+    private primitiveElement!: PrimitiveElementController['primitiveElement'];
+
+    readonly day = input<DateValue>();
+
+    readonly month = input<DateValue>();
+
+    readonly dayValue = computed(() => this.day()?.day.toLocaleString());
+
+    readonly isDateToday = computed(() => {
+        return isToday(<DateValue>this.day(), getLocalTimeZone());
+    });
+
+    readonly isSelectedDate = computed(() => this.rootContext.isDateSelected!(<DateValue>this.day()));
+
+    readonly isDisabled = computed(() => this.rootContext.isDateDisabled!(<DateValue>this.day()));
+
+    readonly isOutsideView = computed(() => {
+        return !isSameMonth(<DateValue>this.day(), <DateValue>this.month());
+    });
+
+    readonly isFocusedDate = computed(() => {
+        return !this.rootContext.disabled && isSameDay(this.day()!, this.rootContext.placeholder());
+    });
+
+    currentElement!: PrimitiveElementController['currentElement'];
+
+    constructor() {
+        const { primitiveElement, currentElement } = usePrimitiveElement();
+
+        this.currentElement = currentElement;
+        this.primitiveElement = primitiveElement;
+    }
+
+    ngAfterViewInit() {
+        this.primitiveElement.set(this.elementRef.nativeElement);
+    }
+
+    protected onClick() {
+        this.changeDate(this.day()!);
+    }
+
+    protected onArrowKey(event: KeyboardEvent) {
+        const code = event.code;
+        if (![
+                'ArrowRight',
```

Comment:
```suggestion
                ARROW_RIGHT,
```
from [kbd-constants](https://github.com/radix-ng/primitives/blob/6e3037e5b852eaae229b43b5b461176f7f80f081/packages/primitives/core/src/kbd-constants.ts)